styles/myStory.css

```diff
@@ -0,0 +1,86 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+}
+.overlay-content {
+  max-width: 1500px;
+  margin: 0 auto;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-left: 20px;
+  height: 90%;
+}
+.overlay-text {
+  position: absolute;
+  color: white;
+  font-family: var(--paragraph-sans);
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.overlay-text p {
+  margin-bottom: 50px;
+  margin: 0px;
+  text-transform: uppercase;
+  text-align: center;
+}
+.text-header {
+  font-size: 68px;
+}
+.text-description {
+  padding-top: 20px;
+  font-size: 16px;
+}
+.color-overlay {
```

Comment:
I don't see the purpose for this?

Reply:
The way that the layout is laid for now. It's also unneccessary to use another absolute overlay. If you want to have a back-up bg-color, you can add it to the same class as the image. So while the image is unavailable, the bg-color will show

Reply:
Ok, I removed the color-overlay